modules/tensor_mechanics/doc/content/source/materials/ADRankTwoCartesianComponent.md

```diff
@@ -0,0 +1,34 @@
+# Rank Two Tensor Component
+
+!syntax description /Materials/ADRankTwoCartesianComponent
+
+## Description
+
+This is an ADMaterial model used to extract components of a rank-2 tensor in a
+Cartesian coordinate system. This can be used regardless of the coordinate
+system used by the model.
+
+This ADMaterial model is used by
+[TensorMechanics/Master](/Modules/TensorMechanics/Master/index.md)
+automatically, but can also be called directly by the user. This class provides
+methods to calculate several different scalar stress ($\boldsymbol{\sigma}$) and
+strain ($\boldsymbol{\epsilon}$)quantities for a Rank-2 tensor, as described in
```

Comment:
Delete this sentence. This just grabs components, and doesn't even use RankTwoScalarTools.

Reply:
RankTwoScalarTools::component is the function used to determine the components.